web/main/background.ts

```diff
@@ -91,17 +96,51 @@ async function waitForGraphQLServer(
   throw new Error("Timed out starting GraphQL server");
 }
 
+function setupTitleBarClickMac() {
+  if (process.platform !== "darwin") {
+    return;
+  }
+
+  ipcMain.on("mac-title-bar-clicked", (event: IpcMainEvent) => {
+    const doubleClickAction = systemPreferences.getUserDefault(
+      "AppleActionOnDoubleClick",
+      "string",
+    );
+    const win = BrowserWindow.fromWebContents(event.sender);
+    if (win) {
+      if (doubleClickAction === "Minimize") {
+        win.minimize();
+      } else if (doubleClickAction === "Maximize") {
+        if (!win.isMaximized()) {
+          win.maximize();
+        } else {
+          win.unmaximize();
+        }
+      }
+    }
+  });
+}
+
 app.on("ready", async () => {
   mainWindow = createWindow("main", {
-    width: 1280,
-    height: 680,
+    width: 1400,
+    height: 900,
+    minHeight: 600,
+    minWidth: 600,
+    titleBarStyle: process.platform === "darwin" ? "hiddenInset" : undefined,
+    titleBarOverlay: process.platform === "darwin",
```

Comment:
What will the app look like on windows? Will it have no title bar at all and lose the nav altogether? Should we use the normal navbar below on windows?